### ISSUE TITLE
Default plotting with multiple metrics

### ISSUE THREAD

**Issue by owlas** [lightdash/lightdash#591]
If I plot:
* 1 dimension
* 2 metrics

The chart defaults to only showing 1 metric. I would have expected to see all my metrics

## Expected

<img width="911" alt="Screenshot 2021-10-12 at 12 04 13" src="https://user-images.githubusercontent.com/11660098/136944306-5b7d53f7-2a18-411d-a04a-a3dcb5ad9eee.png">

## Actual

<img width="912" alt="Screenshot 2021-10-12 at 12 03 48" src="https://user-images.githubusercontent.com/11660098/136944251-e23c4ed9-6680-4b09-8b1f-929514ab070b.png">

**Comment by owlas** [comment]
@TuringLovesDeathMetal do you agree that we should show both metrics?

If only I expect this behaviour then we can close this issue

**Comment by TuringLovesDeathMetal** [comment]
ya, I would expect everything to be plotted that I have in my table by default. 

Then I could remove things if necessary

**Comment by TuringLovesDeathMetal** [comment]
I think there’s one extra thing I’d add (but if it will take more than 1hr, then don’t add it). Here’s the behaviour I’d expect:
1. I open a new explore
2. I add 3 metrics (metrics A, B, C) + 1 dimension
3. I hit run and all three metrics pop up on my chart
4. I unselect all of them except metric A
5. I add another metric: metric D
6. I hit run
7. Now my chart has metric A and metric D appear on my chart.

Basically, whenever you add a new metric or table calculation it should be automatically plotted. So, unless you unselect the metric/table calculation at some point, it should be added as a series by default.

**Comment by owlas** [comment]
Closing in favour of #1607
